look.go

```diff
@@ -431,13 +431,15 @@ func includename(t *Text, r string) string {
 	return runestr(r, n);
 }
 */
+
+// Dirname returns the directory name of the path in the tag file of t.
```

Comment:
nit: needs to be `dirname` for `godoc` yes?

Reply:
You're right. Somehow I thought the convention was to capitalize first letter of the sentence, as in Plan 9 man pages. It's unexported, so it doesn't really matter.